webapp/templates/jury/contest.html.twig

```diff
@@ -356,6 +357,13 @@
                                     </a>
                                 {% endif %}
                             </td>
+                            <td>
+                                {% if is_granted('ROLE_ADMIN') %}
+                                    <a href="{{ path('jury_contest_problem_request_remaining', {'contestId': contest.cid, 'probId': problem.probid}) }}">
```

Comment:
We should probably add a `title="Judge remaining` here (and similar with the delete button above) to have a hover text.

Reply:
Why does the W3||WCAG test not flag this. I'll fix it.